packages/twenty-front/.storybook/preview.tsx

```diff
@@ -8,6 +8,7 @@ import { RootDecorator } from '../src/testing/decorators/RootDecorator';
 import { mockedUserJWT } from '../src/testing/mock-data/jwt';
 
 import 'react-loading-skeleton/dist/skeleton.css';
+import 'twenty-ui/style.css';
```

Comment:
The way the build is working:
- storybook vite build will find the vite.config.ts in front folder and use it
- vite.config.ts will require to pre-build linaria style and add a <link> to a css in index.html
- however, as we twenty-ui is now pre-built, we need to include it manually here